scipy/stats/tests/test_distributions.py

```diff
@@ -3074,41 +3188,32 @@ def test_fit(self):
         assert_almost_equal(scale2, .02503, 4)
         assert_almost_equal(loc2, .03354, 4)
 
-    @pytest.mark.slow
-    def test_pdf_nolan_samples(self):
-        """ Test pdf values against Nolan's stablec.exe output
-            see - http://fs2.american.edu/jpnolan/www/stable/stable.html
-
-            There's a known limitation of Nolan's executable for alpha < 0.2.
-
-            The data table loaded below is generated from Nolan's stablec
-            with the following parameter space:
-
-                alpha = 0.1, 0.2, ..., 2.0
-                beta = -1.0, -0.9, ..., 1.0
-                p = 0.01, 0.05, 0.1, 0.25, 0.35, 0.5,
-                    and the equivalent for the right tail
-
-            Typically inputs for stablec:
-
-                stablec.exe <<
-                1 # pdf
-                1 # Nolan S equivalent to S0 in scipy
-                .25,2,.25 # alpha
-                -1,-1,0 # beta
-                -10,10,1 # x
-                1,0 # gamma, delta
-                2 # output file
-        """
-        data = np.load(
-            Path(__file__).parent /
-            'data/levy_stable/stable-Z1-pdf-sample-data.npy'
-        )
-
-        data = np.core.records.fromarrays(data.T, names='x,p,alpha,beta,pct')
-
-        # support numpy 1.8.2 for travis
-        npisin = np.isin if hasattr(np, "isin") else np.in1d
```

Comment:
Was this check for `np.isin` unnecessary? The comment makes it seem like it was removed to handle a CI quirk, but it may no longer be needed.

Reply:
I think it's unnecessary now. The latest scipy requires numpy >= 1.16.5, so supporting 1.8.2 should no longer be necessary and Travis isn't even used for CI any longer.